apiary-metastore-events/apiary-metastore-consumer/README.md

```diff
@@ -0,0 +1,14 @@
+# Overview
+
+`apiary-hive-consumers` provides implementation of features that are relying on the Hive Metastore events.
+  
+ Each of the consumers consists of:
+  - SQS Queue (retrieves/filters messages from SNS)
```

Comment:
doesn't really consist of this, is probably captured in the `SQS Queue Consumer`

Reply:
Perhaps instead of doing consist of list, I can add to the sentence above something like: 
Each of the consumers has own SQS Queue that contains only relevant to it events ?

Reply:
ok yeah, reword that a bit but I think I know what you mean.